src/main/clm_varctl.F90

```diff
@@ -196,6 +196,9 @@ module clm_varctl
   logical, public :: use_fates = .false.            ! true => use fates
 
   ! These are INTERNAL to the FATES module
+  integer, public            :: fates_parteh_mode = -9                 ! 1 => carbon only
```

Comment:
@rgknox what does the -9 mean here?

Reply:
it is a bogus initialization value, that will force the model to fail if not set correctly.

Reply:
ok, let's add a comment to that effect in case its not obvious.

```suggestion
  integer, public            :: fates_parteh_mode = -9                 ! 1 => carbon only; -9 => junk value that will force fail unless reset later
```